docs/update-prod-data.md

```diff
@@ -0,0 +1,82 @@
+# Steps to update the dev db with prod data
+
+
+## 1 - Take a Dump of Data from Production and Export to a File
+
+You'll first need to take a dump of the data from the production environment and export it to a file. This file will contain all of the data that is currently in the production database.
+
+You will first have to use **kubectl** to get the list of available pods `kubectl get pod`
```

Comment:
Which is first? get a db dump? or use kubectl? See my comment further below. 

Remove adverbs like "first(ly)" "second(ly)" "after" "finally" "now" since they're filler. Everything is already numbered.